manual/README.md

```diff
@@ -65,7 +65,8 @@ The following setup assumes a Unix-like environment.
 
 6. Initialize the node:
    ```shell
-   export MEZOD_CHAIN_ID=mezo_31611-1 # Mezo testnet
+   # Use mezo_31611-1 as chain ID for testnet or mezo_31612-1 for mainnet.
+   export MEZOD_CHAIN_ID= ...
```

Comment:
I propose to use `mainnet` configuration in all examples as it is for `docker` deployment.
```suggestion
   export MEZOD_CHAIN_ID=mezo_31612-1
```